src/current/_includes/releases/v24.1/v24.1.4.md

```diff
@@ -0,0 +1,98 @@
+## v24.1.4
+
+Release Date: August 29, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-4-security-updates">Security updates</h3>
+
+- URLs in the following SQL statements are now sanitized of any secrets before being written to [unredacted logs]({% link v24.1/configure-logs.md %}#redact-logs). [#127360][#127360]
+- [`ALTER BACKUP SCHEDULE`]({% link v24.1/alter-backup.md %})
+- [`ALTER BACKUP`]({% link v24.1/alter-backup.md %})
+- [`ALTER CHANGEFEED SET sink`]({% link v24.1/alter-changefeed.md %})
+- [`BACKUP`]({% link v24.1/backup.md %})
+- [`COPY`]({% link v24.1/copy-from.md %})
+- [`CREATE CHANGEFEED`]({% link v24.1/create-changefeed.md %})
+- [`CREATE EXTERNAL CONNECTION`]({% link v24.1/create-external-connection.md %})
+- [`CREATE SCHEDULE FOR BACKUP`]({% link v24.1/create-schedule-for-backup.md %})
+- [`CREATE SCHEDULE FOR CHANGEFEED`]({% link v24.1/create-schedule-for-changefeed.md %})
+- [`EXPORT`]({% link v24.1/export.md %})
+- [`IMPORT INTO`]({% link v24.1/import-into.md %})
+- [`RESTORE`]({% link v24.1/restore.md %})
+- [`SHOW BACKUPS`]({% link v24.1/show-backup.md %})
+- [`SHOW BACKUP`]({% link v24.1/show-backup.md %})
+
+<h3 id="v24-1-4-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Added a new Kafka [changefeed sink]({% link v24.1/changefeed-sinks.md %}) that uses the [`franz-go` library](https://github.com/twmb/franz-go) and CockroachDB's `batching_sink` implementation. The new Kafka sink can be enabled with the `changefeed.new_kafka_sink_enabled` [cluster setting]({% link v24.1/cluster-settings.md %}), which is disabled by default. [#128018][#128018]
+- The new Kafka sink, enabled with [`changefeed.new_kafka_sink_enabled`]({% link v24.1/cluster-settings.md %}), as well as the PubSub sink, now display notices indicating the topics that a Changefeed will emit to. [#128333][#128333]
+
+<h3 id="v24-1-4-sql-language-changes">SQL language changes</h3>
+
+- Added an option to prevent the [`GRANT OPTION`]({% link v24.1/show-grants.md %}#privilege-grants) from being given to the owner of an object by default. The new `sql.auth.grant_option_for_owner.enabled` cluster setting defaults to `TRUE`, retaining the existing behavior, and when it is set to `FALSE`, the `GRANT OPTION` is not implicitly given to an object's owner. The owner still has all privileges on an object except the ability to grant privileges to other users. [#126959][#126959]
```

Comment:
```suggestion
- Added a new `sql.auth.grant_option_for_owner.enabled` [cluster setting]({% link v24.1/cluster-settings.md %}) to prevent the [`GRANT OPTION`]({% link v24.1/show-grants.md %}#privilege-grants) from being given to the owner of an object by default. The cluster setting defaults to `true`, retaining the existing behavior; when it is set to `false`, the `GRANT OPTION` is not implicitly given to an object's owner. The owner will still have all privileges on an object except the ability to grant privileges to other users. [#126959][#126959]
```